### DISCUSSION TITLE
Feature request: nameable MIDI and CV tracks

### DISCUSSION THREAD

**Discussion by miaouxmiaoux** [SynthstromAudible/DelugeFirmware#910] [Ideas]
It would be great to be able to do this!

**Comment by saynotohugs** [comment]
It would mean SO much :)

**Comment by m-m-adams** [comment]
MIDI tracks can now be named+saved (save+midi, load+midi). I'm not sure of the utility of doing it for CV tracks since the name always matches the CV out currently?

**Comment by miaouxmiaoux** [comment]
That's great! Thank you

**Comment by patriciachadwick62** [comment]
THANK Y
would be nice if the cc could be Labled too            and saved with in the File
(push golden encoder + (after selectet) push select to name)

and would be verry helpfull if there would be possible to save the standart value with

so think
a menue with                       name CC
and.                                      norm value

on                                          push golden encoder + push select encoder

Thank Y

**Comment by saynotohugs** [comment]
Wonderful that we can now name midi tracks!! 
Thanks so much :)
